source/defclass-star.lisp

```diff
@@ -64,6 +64,10 @@ subtypes, e.g.  for \"\" return 'string instead of `(SIMPLE-ARRAY CHARACTER
 Note that in a slot definition, '() is inferred to be a list while NIL is
 inferred to be a boolean.
 
+If the slot is found in a finalized superclass, the inferred type is then that
+of the class.  If some superclass is not finalized, no type inference is
```

Comment:
"of the superclass"?

Reply:
Where exactly?

Reply:
"If the slot is found in a finalized superclass, the inferred type is then that of the _superclass_."

Reply:
Oh right!